uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api.spec.js

```diff
@@ -584,6 +605,10 @@ define([
       $httpBackend.whenGET(/action=getattachments&entity=LeaveRequest/)
         .respond(mockData.getAttachments());
 
+      // Intercept backend calls for LeaveRequest.getBreakdown
+      $httpBackend.whenGET(/action=getBreakdown&entity=LeaveRequest/)
+        .respond(mockData.all());
```

Comment:
Shouldn't this responde with the breakdown mock data?

Reply:
Oh. Apologies!